Testscripts/Linux/VERIFY-LIS-MODULES-VERSION.sh

```diff
@@ -42,6 +42,7 @@ skip_modules=()
 config_path="/boot/config-$(uname -r)"
 if [[ $(detect_linux_distribution) == clear-linux-os ]]; then
     config_path="/usr/lib/kernel/config-$(uname -r)"
+    LogMsg "Set the configuration path to $config_path"
```

Comment:
Put this line outside if?

Reply:
Fixed in 69aa5c4f471226535ca1a0bcb9683c41efe546b4